### DISCUSSION TITLE
Blocking a user on github

### DISCUSSION THREAD

**Discussion by codermal7** [community/community#139128] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Does, blocking a user on GitHub prevents them from seeing my profile ?

**Comment by Sanjaya-Samudra** [comment]
Blocking a user on GitHub primarily does the following:

1. **Profile Visibility**: The blocked user cannot view your GitHub profile.

2. **Interactions**: The blocked user cannot comment on your repositories, open issues, or pull requests, and they cannot send you messages.

3. **Notifications**: You won’t receive notifications from the blocked user.

4. **Collaboration**: If the blocked user tries to collaborate on any of your projects, they will be restricted.

However, the blocked user can still see your public repositories and any contributions that are publicly visible. The block is more about restricting direct interactions rather than hiding your work.

**Reply by jidanni** [reply]
What's worse is you can still comment on issues the blocked user created, but they won't get notified you made a comment.

Which makes it look like they saw what you wrote but had no opinion.

Sort of like when one party does not show up in a courtroom.
